Tests/Entity/CategoryManagerTest.php

```diff
@@ -21,6 +21,7 @@ public function testGetPager()
         $self = $this;
         $this
             ->getCategoryManager(function ($qb) use ($self) {
+                $qb->expects($self->once())->method('getRootAliases')->will($self->returnValue(array()));
```

Comment:
`$qb->expects($self->any())->method('getRootAliases')->will($self->returnValue(array()));` to make it build on `--prefer-lowest` :)